swarm/storage/resource.go

```diff
@@ -30,6 +30,25 @@ const (
 	resourceHash        = SHA3Hash
 )
 
+type blockEstimator struct {
+	Start   time.Time
+	Average time.Duration
+}
+
+func NewBlockEstimator() *blockEstimator {
+	ropstenStart, _ := time.Parse(time.RFC3339, "2016-11-20T11:48:50Z")
+	return &blockEstimator{
+		Start:   ropstenStart,
+		Average: 14467100 * time.Microsecond,
```

Comment:
is this value calculated from real data?

Reply:
yes, from etherscan.io